### PR TITLE
Add documentation regarding SystemVMs load-balancing and SSL-Offloading

### PR DESCRIPTION
**Pull request by sbrueseke**
I found it very difficult to get load-balancing for Console Proxy and Secondary Storage VM working and added some information regarding a working SSL-Offloading.

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

**Review by DaanHoogland — COMMENTED** [review]
Looks good, thanks Sven. some minor comments

**Comment by sbrueseke** [comment]
@DaanHoogland thx Daan! I did some layout fixing so the port reference will be shown in a list. Hope it made it to the pull request.

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

File: source/adminguide/systemvm.rst [hunk 1]
```diff
@@ -372,17 +372,43 @@ are still in default PEM format (no URL encoding needed here).
...
+proxy's IP address behind the FQDN. When using a load balancer it is
+also possible to perform SSL-Offloading, so no certificate needs to be
+configured on CloudStack itself. For further information please see
 https://cwiki.apache.org/confluence/display/CLOUDSTACK/Realhost+IP+changes
 for more details.
 
+These ports needed to be configured for load-balancing:
+-  443 to 443 (to CPVM)
+-  8080 to 8080 (to CPVM)
+-  443 to 443 (to SSVM)
```
Comment (DaanHoogland) [inline source/adminguide/systemvm.rst:390]:
```suggestion
These ports needed to be configured for load-balancing:

- 443 to 443 (to CPVM)
- 8080 to 8080 (to CPVM)
- 443 to 443 (to SSVM)
```

File: source/adminguide/systemvm.rst [hunk 2]
```diff
@@ -372,17 +372,43 @@ are still in default PEM format (no URL encoding needed here).
...
+
+SSL-Offloading with Load-balancing for Console Proxies / Secondary Storage VMs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To implement SSL-Offloading you need 2 public IP addresses (one for Console Proxy and one for Secondary Storage VM) which
+each of them resolve to a different FQDN and terminate at the load balancer. Also 3 global settings need to be edited.
+- The setting ‘consoleproxy.url.domain’ to the FQDN used by the certificate (For example: cpvm.company.com)
+- The setting ‘secstorage.ssl.cert.domain’ to the FQDN used by the cerrificate (For example: ssvm.company.com)
+- The setting ‘secstorage.encrypt.copy’ to true
```
Comment (DaanHoogland) [inline source/adminguide/systemvm.rst:398]:
```suggestion
each of them resolve to a different FQDN and terminate at the load balancer. Also 3 global settings need to be edited.

- The setting ‘consoleproxy.url.domain’ to the FQDN used by the certificate (For example: cpvm.company.com)
- The setting ‘secstorage.ssl.cert.domain’ to the FQDN used by the cerrificate (For example: ssvm.company.com)
- The setting ‘secstorage.encrypt.copy’ to true
```

File: source/adminguide/systemvm.rst [hunk 3]
```diff
@@ -372,17 +372,43 @@ are still in default PEM format (no URL encoding needed here).
...
+   possible to add a fake public IP range to CloudStack which uses internal IP addresses for SystemVM use only. Please
+   be aware that the load balancer needs access to the used IP addresses to forward traffic.
+
+After edited global settings mentioned above you need to recreate both System VMs by destroying them. CloudStack will recreate
+them with the new settings automatically.
+
+When using SSL-Offloading you need to configure following ports on the load balancer after adding the correct certificate to the public IP of each FQDN:
+-  lb-publicip1:443 to CPVM:80
+-  lb-publicip1:8080 to CPVM:8080
+-  lb-publicip2:443 to SSVM:80
```
Comment (DaanHoogland) [inline source/adminguide/systemvm.rst:411]:
```suggestion
When using SSL-Offloading you need to configure following ports on the load balancer after adding the correct certificate to the public IP of each FQDN:

- lb-publicip1:443 to CPVM:80
- lb-publicip1:8080 to CPVM:8080
- lb-publicip2:443 to SSVM:80
```

**Comment by DaanHoogland** [comment]
@sbrueseke if you go to files you can add all sugestions that you want in one commit. I _think_ my latest suggestions will solve your list issue.

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

**Comment by sbrueseke** [comment]
LGTM

**Pull request merged**